src/test/java/ru/yandex/qatools/embed/postgresql/TestDownloads.java

```diff
@@ -18,6 +18,9 @@
     public void testDownloads() throws IOException {
         IArtifactStore artifactStore = new PostgresArtifactStoreBuilder().defaults(Command.Postgres).build();
 
+        if(true){
```

Comment:
Use the @Ignore annotation from JUnit - so in the logs you will see the disabled tests